### PR TITLE
Unify description of unpack instructions

### PR DESCRIPTION
**Pull request by yulyugin**
* Fix operand size for UNPCKHPS
* Introduce SKIPHIGH64 similar to SKIPLOW64 used by 'high packed' unpack instructions.

### CODE REVIEW

File: datafiles/xed-isa.txt [hunk 1]
```diff
@@ -8861,7 +8861,7 @@ ATTRIBUTES : SKIPLOW64  REQUIRES_ALIGNMENT
 PATTERN   : 0x0F 0x15 no_refining_prefix MOD[mm] MOD!=3 REG[rrr] RM[nnn]  MODRM()
 OPERANDS  : REG0=XMM_R():rw:ps MEM0:r:dq
 PATTERN   : 0x0F 0x15 no_refining_prefix MOD[0b11] MOD=3 REG[rrr] RM[nnn]
-OPERANDS  : REG0=XMM_R():rw:ps REG1=XMM_B():r:dq
+OPERANDS  : REG0=XMM_R():rw:ps REG1=XMM_B():r:q
```
Comment (markcharney) [inline datafiles/xed-isa.txt:8863]: Gonna reject  this one. Another developer saw  this and also raised concerns.  The way XED  is expressing  this is as if 128b of the  reg1 operand  were read, but the SKIPLOW64  is there to  tell you  only the upper 64b  are referenced.  I  see what you  are  trying to  say -- q  is more  accurate than  dq  if you are thinking about  how much data  are referenced. The  intent  of  SKIPLOW64  was to communicate the same thing as what you  are trying  to get at.

**Comment by markcharney** [comment]
and  please  do  not  delete  my comments.

File: datafiles/xed-isa.txt (hunk 1 above)
Comment (yulyugin) [reply datafiles/xed-isa.txt:8863]: Sure, no problem but then we will need to change operand sizes for other instructions with SKIPLOW64. They have 'q' at the moment while only this one has 'dq'. BTW, what's about MOVH there you have 'q' as well while high part is getting read. Should we introduce 'hq' or something similar then?

**Comment by yulyugin** [comment]
What about the SKIPHIGH64 part? Is it okay. Comments are much harder to parse than such and attributes.

**Comment by markcharney** [comment]
what is the benefit of SKIPHIGH64?

**Comment by markcharney** [comment]
Dude, send me email  internally. I closed this.

**Pull request closed without merging**